src/core/bpf-lsm.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include "hashmap.h"
+
+typedef struct Unit Unit;
+typedef struct Manager Manager;
+
+typedef struct restrict_fs_bpf restrict_fs_bpf;
+
+int lsm_bpf_supported(void);
+int lsm_bpf_setup(Manager *m);
+int bpf_restrict_filesystems(Unit *u, const Set *filesystems, bool allow_list);
+int cleanup_lsm_bpf(const Unit *u);
```

Comment:
hmm, most other symbols here use `lsm_bpf` as prefix, not suffix. do so here too? (and in fact for al symbols?

the ones that take a Unit* as first arg might even use `lsm_bpf_unit_` as prefix?

Reply:
Makes sense, I'll make them all consistent.